### ISSUE TITLE
Backup destination configuration

### ISSUE THREAD

**Issue by ezequiel** [jlevy/repren#2]
First off, thanks for the great tool. 

I like the fact it auto-creates backups. The only thing it seems to be missing is the ability to configure where backups are stored. Is this currently possible?

**Comment by jlevy** [comment]
No, at the moment it's hard-coded to ".orig". Making the suffix customizable would be easy. Feel free to give a PR, or I'll add it in the next version.

**Comment by ezequiel** [comment]
Sure. I will send a pull request.
